Rakefile

```diff
@@ -318,7 +363,7 @@ def create_anonymous_user
       anonymous_login = config['login']
       anonymous_password = config['password']
 
-      if anonymous_login and anonymous_password and not ScalarmUser.find_by_login(anonymous_login)
+      if anonymous_login and anonymous_password and not ScalarmUser.where(login: anonymous_login).first
```

Comment:
To zostało dodane przypadkowo. Mozna zostawić jako ze to wywołanie jest równoważne a mozna tez wykasować jesli jest ku temu jakis powód.